### PR TITLE
Update incident type

### PR DESCRIPTION
**Pull request by tjbburton**
- Update mapping for incident type to allow multiple answers
- Refactor injury rules into their own functions
- Add integration tests

### CODE REVIEW

**Comment by tjbburton** [comment]
Proposed version patch

File: function/services/eligibility-checker/index.js [hunk 1]
```diff
@@ -4,6 +4,38 @@ const {DateTime} = require('luxon');
...
+            }
+        });
+    }
+    return ineligibleDueToInjuries;
+}
+
+function checkEligibilityForNoInjuries(applicationFormJson) {
+    let noInjuries = false;
+    const applicationType = applicationFormJson?.application_type;
+    if (applicationType === 2 || applicationType === 3) {
```
Comment (neil-stephen-mcgonigle) [inline function/services/eligibility-checker/index.js:28]: Replace with meaningfull constants for applicationType

Comment (tjbburton) [reply function/services/eligibility-checker/index.js:28]: Happy to do this, do you know where the different application types are stored so I can get them right?

Comment (neil-stephen-mcgonigle) [reply function/services/eligibility-checker/index.js:28]: Sorry just saw this, no I don't but I will have a look

**Review by neil-stephen-mcgonigle — APPROVED** [review]

**Pull request merged**